src/Scripting/VisualBasic/Hosting/ObjectFormatter/VisualBasicPrimitiveFormatter.vb

```diff
@@ -0,0 +1,81 @@
+' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Scripting.Hosting
+Imports Microsoft.CodeAnalysis.Scripting.Hosting.ObjectFormatterHelpers
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Scripting.Hosting
+
+    Public Class VisualBasicPrimitiveFormatter
+        Inherits CommonPrimitiveFormatter
+
+        Protected Overrides ReadOnly Property NullLiteral As String
+            Get
+                Return "Nothing"
+            End Get
+        End Property
+
+        Protected Overrides Function FormatLiteral(value As Boolean) As String
+            Return ObjectDisplay.FormatLiteral(value)
+        End Function
+
+        Protected Overrides Function FormatLiteral(value As Date) As String
+            Return ObjectDisplay.FormatLiteral(value)
+        End Function
+
+        Protected Overrides Function FormatLiteral(value As String, useQuotes As Boolean, escapeNonPrintable As Boolean, Optional numberRadix As Integer = NumberRadixDecimal) As String
```

Comment:
I'm beginning to second guess this a bit...should ```FormatLiteral``` have argument validation and throw for combinations that won't be honored by VB ObjectDisplay.FormatLiteral?  (for both ```string``` and ```char``` overloads)